app/lib/coursesHelper.coffee

```diff
@@ -225,12 +240,11 @@ module.exports =
     labels = []
     courseLabelIndexes = CS: 0, GD: 0, WD: 0
     for course in courses
-      acronym = switch
-        when /game-dev/.test(course.get('slug')) then 'GD'
-        when /web-dev/.test(course.get('slug')) then 'WD'
-        else 'CS'
+      acronym = courseShortName(course)
```

Comment:
In the future could fully refactor this function to not store a counter.
No regression tests in place.